src/features/projects/Settings.tsx

```diff
@@ -290,6 +297,62 @@ export function Settings() {
               </dd>
             </dl>
           </div>
+          <div className="section setting_box" id="advanced">
+            <div className="setting_title">
+              <strong className="text">Advanced</strong>
+            </div>
+            <dl className="sub_info">
+              <dt className="sub_title">Client Deactivate Threshold</dt>
+              <dd className="sub_desc">
+                <div
+                  className={classNames('input_field_box', {
+                    is_error: checkFieldState('clientDeactivateThreshold', 'error'),
+                    is_success: checkFieldState('clientDeactivateThreshold', 'success'),
+                  })}
+                >
+                  <InputTextField
+                    reset={() => {
+                      resetForm();
+                      resetUpdateFieldInfo();
+                    }}
+                    {...register('clientDeactivateThreshold', {
+                      required: 'Client Deactivate Threshold is required',
+                      pattern: {
+                        value: /^(\d{1,2}h\s?)?(\d{1,2}m\s?)?(\d{1,2}s)?$/,
+                        message:
+                        'Client Deactivate Threshold should be signed sequence of decimal numbers, each with optional fraction and a unit suffix, such as "23h30m10s", "-1.5h" or "2h45m"',
```

Comment:
The value `-1.5h` in the message is not an appropriate.
I have some questions:
- Is it okay to start with 0, such as `02h`?
- If I set value to `01s`, will the client be deactivated after one second?

Reply:
Thank you for letting me know.
Removed `-1.5h`

For your questions:
- It is okay to start with 0. You can test this regex on here: https://regex101.com/r/1i8udR/1
- Yes, client will be deactivated after one second if value is set to `01s`. Currently there are no limits on this value. I'm will add minimum limit on server & client on further updates.

Reply:
Thank you for answering.
Similarly, it would be better to remove the term `optional fraction`.

Reply:
`optional fraction` seems unnecessary for me too.
Removed `optional fraction`.